clarification on the use of !reset (#373)
compose-spec/compose-spec: 13-merge.md

### Reset value

In addition to the previously described mechanism, an override Compose file can also be used to remove elements from your application model.
For this purpose, the custom [YAML tag](https://yaml.org/spec/1.2.2/#24-tags) `!reset` can be set to
override a value set by the overriden Compose file. A valid value for attribute must be provided,
but will be ignored and target attribute will be set with type's default value or `null`. 

> For readability it is recommended to set attribute value to null value (`null`) or empty array `[]`
> when using `!reset` whenever value is actually unused, so that it is explicit resulting attribute 
> will be set to null.

Reviewer: ```suggestion
For readability, it is recommended to explicitly set the attribute value to the null (`null`) or empty array `[]` (with `!reset null` or `!reset []`)
so that it is clear that resulting attribute will be cleared.
```
